### ISSUE TITLE
Scheduled Items outside of workflow are cancelled if an item in workflow has status of resubmitted or rejected

### ISSUE THREAD

**Issue by KevinJump** [nathanwoulfe/Plumber#88]
**Describe the bug**
Scheduled Publishes are cancelled, even if the node isn't in workflow, I think this is because the repo contacts the statuses to the SQL query without brackets (so one matching status is enough to find something)

The check in the ContentService Publishing event - will stop a publish if anything comes back from `GetForNodeByStatus` which is eventually calling the repository here:

https://github.com/nathanwoulfe/Plumber/blob/2dbf533efd37a94a7ca7f9f5fe234690b1c55a59/Workflow/Repositories/InstancesRepository.cs#L137

this concats the Status - so the actuall query you get is . 

SELECT * FROM WorkflowInstance where nodeId = 32973 AND STATUS = 3 OR STATUS = 2 OR STATUS = 7;

This returns something when any item has a status of 2 or 7. ideally this should be ?

SELECT * FROM WorkflowInstance where nodeId = 32973 AND ( STATUS = 3 OR STATUS = 2 OR STATUS = 7) ;

_i think_

**To Reproduce**
1. Have at least one item in the status of Rejected or Resubmitted 
2. Schedule a item that isn't in workflow
3. Umbraco Logs will show : 
`Could not published the document (32973) based on it's scheduled release. Status result: FailedCancelledByEvent`

4. With Debugging on you will see: 
```
INFO  Workflow.Events.Handlers.BeforePublish - Scheduled publish for TEST NEWS ITEM (SCHEDULE) cancelled due to active workflow
INFO  Umbraco.Core.Publishing.PublishingStrategy - Content 'TEST NEWS ITEM (SCHEDULE)' with Id '32973' will not be published, the event was cancelled.
```
**Plumber version**
v1.0.4

**Comment by nathanwoulfe** [comment]
@KevinJump that's a ridiculously good find. I'll get on it.

**Comment by nathanwoulfe** [comment]
Spot on with the SQL construction - adding parenthesis fixes this. Does expose a bug though on the info tab where the latest status is saved and published, despite being cancelled. Need to check if this is my doing, or an Umbraco bug.

**Comment by nathanwoulfe** [comment]
Resolved in v1.1.0
